clouddriver-lambda/src/main/java/com/netflix/spinnaker/clouddriver/lambda/deploy/description/CreateLambdaFunctionConfigurationDescription.java

```diff
@@ -30,4 +34,11 @@
   String role;
   String runtime;
   Integer timeout;
+  List<String> subnetIds;
+  List<String> securityGroupIds;
+  Environment environment;
+  DeadLetterConfig deadLetterConfig;
+  String encryKMSKeyArn;
```

Comment:
can you make this encryptionKMSKeyArn (the abbreviated version looks strange / like a typo to me)

Reply:
Implemented.